### PR TITLE
added azimuthpointer and updated pin popups

### PR DESCRIPTION
**Pull request by CoolAEW**
Added a pointer to markers on nodes that have azimuth data.
Updated the popups with azimuth, elevation and height data.

image

### CODE REVIEW

**Comment by aanon4** [comment]
So you will need to rebase your changes from my current repo, which I've been working on today (minutes ago in fact) and there are a few changes which conflict.

Also, I'm assuming you've not actually changed the package.json and pacakge-lock.json files? Please dont check anything in which isnt an explicit change for this feature.

**Comment by CoolAEW** [comment]
I have in fact changed package.json and package-lock.json to add `leaflet-rotatedmarker` which I needed to rotate the arrow around the markers. The version used is compatible with the leaflet plugin that is already in use.

**Comment by aanon4** [comment]
Okay; could you make sure that file reflects just the change you made to it. It appears to have been reformatted/re-whitespaces making your change impossible to see.

File: src/components/BaArednMap.jsx [hunk 1]
```diff
@@ -182,10 +183,10 @@ class BaArednMap extends Component {
                 rfdtdconns.push(conn);
               }
               break;
-            case 'XLINK':
+            case 'BB':
```
Comment (aanon4) [inline src/components/BaArednMap.jsx]: This needs to remain XLINK.

File: src/components/BaArednMap.jsx [hunk 2]
```diff
@@ -182,10 +183,10 @@ class BaArednMap extends Component {
...
               rfdtdconns.push(conn);
               break;
-            case 'SUPER':
+            case 'STUN':
```
Comment (aanon4) [inline src/components/BaArednMap.jsx]: This needs to remain SUPER

**Review by aanon4 — CHANGES_REQUESTED** [review]

**Comment by aanon4** [comment]
The popup does not appear to handle the situation when the new value (azimuth, etc) are missing.

**Comment by CoolAEW** [comment]
This got messy, so I started over and made a new PR.
I have fixed the popup to only show data that is available.

**Pull request closed without merging**